src/OpenTelemetry/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@
 * Removed the dependency on System.Reflection.Emit.Lightweight
   ([#4140](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4140))
 
+* The `AddOpenTelemetry` extension will now register an `IHostedService` if
```

Comment:
lets do a more detailed write up here for folks updating to this version. Most end users do not really have the context about this, so we should write the steps one should follow when updating to this version.
1. Remove the nuget reference to otel.ext.hosting.
2. Remove the startwithhost(). 
3. (change ns based on the other PR from alan.)

Its okay to do this in a separate PR.

Reply:
also need to write the steps for those users who have to manually retrieve MP/TP .

Reply:
I will tackle this in a follow-up. Need to update documentation and then I'll drop a link here for more info.